M2/Macaulay2/packages/SuperLinearAlgebra.m2

```diff
@@ -0,0 +1,758 @@
++newPackage(
```

Comment:
@FatemehTarashi you need to remove this extra "+" sign from beginning of this file, that's why it's failing.

Reply:
Thank you. ☺️